lesson05/assignment/css/style.scss

```diff
@@ -4,3 +4,161 @@
 @import 'settings.responsive';
 @import 'components.content';
 
+// Colors & fonts
+
+$almostblack: #2a1f2d;
+$dkbrown: #3b2c35;
+$sage: #5b6c5d;
+$mauve: #b098a4;
+$pinkgray: #dbd9db;
+
+$font-text-fancy: 'Architects Daughter', cursive;
+$font-text-plain: Arial, sans-serif;
+$font-header: sunshine, sans-serif;
+
+@mixin no-m-b-p {
+  margin: 0;
+  border: 0;
+  padding: 0;
+}
+
+@mixin no-m-b {
+  margin: 0;
+  border: 0;
+}
+
+// Grid setup
+
+body {
+  box-sizing: border-box;
+  @include no-m-b-p;
+    display: grid;
+    grid-gap: 0;
+    grid-template-areas: 'title'
```

Comment:
Your grid looks fantastic! You now have a good structure in place you could use in your project.